include/vapor/Grid.h

```diff
@@ -88,6 +88,10 @@ class VDF_API Grid {
     //!
     const std::vector<size_t> &GetDimensions() const { return (_dims); }
 
+    //! Get and set the default Z value for a grid.
+    auto GetDefaultZ() const -> double {return _defaultZ;};
+    void SetDefaultZ( double d)        {_defaultZ = d;};
```

Comment:
You have two different techniques of defining return values right next to each other

Reply:
Not sure what you mean, but one of these 2 functions has no return value.

Reply:
void is a return type.

Reply:
I'm not sure if you're suggesting a change or not. It's just not valid syntax to write something like `auto func() -> void`.

Reply:
`auto func() -> void` is valid syntax, however the syntax is inconsistent with 99% of Vapor's code base so I am suggesting we don't use it.

Reply:
1) are you sure? 2) well, any new language feature will be 99% inconsistent with existing code base. 

It's a topic that can be discussed to whether add to the VAPOR style convention a certain syntax, but currently the style convention doesn't say one way or another.

Reply:
@shaomeng will change and we will update the convention